pyvista/examples/__init__.py

```diff
@@ -4,3 +4,4 @@
 from pyvista.examples.examples import *
 from . import gltf
 from . import hdr
+from . import vrml
```

Comment:
We're not adding in the examples in our documentation at the moment. Let's add that in so they get rendered and we can see these new VRML examples in the documentation.

Reply:
That's a good idea. I'll add it in this PR.